src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java

```diff
@@ -78,6 +78,11 @@ protected void registerDefaultRecipes(@Nonnull List<ItemStack> recipes) {
         recipes.add(new ItemStack(Material.QUARTZ_BLOCK));
         recipes.add(new ItemStack(Material.QUARTZ, 4));
 
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
```

Comment:
Version check no longer required 😅 

```suggestion
        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
```